a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -445,10 +469,30 @@ def _get_patched_ve_ip(self, ve_ip):
         return a10_utils.merge_host_and_network_ip(self._subnet.cidr, ve_ip)
 
     @device_context_switch_decorator
-    def delete_device_vlan(self, vlan_id, subnet_id, device_id=None, default_device_id=None, project_id=None):
+    def check_ve_ip_exists(self, vlan_id, config_ve_ip):
+        if self.axapi_client.vlan.exists(vlan_id):
+            if self.axapi_client.interface.ve.exists(vlan_id):
```

Comment:
No point in checking that it exists before getting it. Better to just handle the potential error here so that there isn't 2 of the the exact same get calls made unnecessarily.